spacelift/resource_stack.go

```diff
@@ -397,10 +439,20 @@ func stackInput(d *schema.ResourceData) structs.StackInput {
 				StackName: toString(pulumi[0].(map[string]interface{})["stack_name"]),
 			},
 		}
-	} else {
-		ret.VendorConfig = &structs.VendorConfigInput{
-			Terraform: &struct{}{},
+	} else if terraform, ok := d.Get("terraform").([]interface{}); ok && len(terraform) > 0 {
```

Comment:
I can't comment on this, but don't we want the above 
```
if terraformVersion, ok := d.GetOk("terraform_version"); ok {
		ret.TerraformVersion = toOptionalString(terraformVersion)
	}
```

to set the vendor specific config?

Reply:
So we can get rid of the top-level one from the API

Reply:
Done, I think.